src/Jupyter/res/kernel.js

```diff
@@ -18,7 +18,7 @@ define(
                         {
                             token: "comment",
                             // include % to support kata special commands
-                            regex: /(\/\/|%).*/
+                            regex: /(\/\/|%kata).*/
```

Comment:
We'll need to list all our special commands if we are to support them individually... Off the top of my head I remember `%package`, `%workspace` and `%check_kata`